utils/normalize/myList.js

```diff
@@ -0,0 +1,15 @@
+import { normalize, schema } from 'normalizr';
+
+export default (data) => {
+  const movie = new schema.Entity('listMovies');
```

Comment:
You need to have the same key for same entities. For example, if I want normalize json with movies, I will use `movies` and will get the same key for all normalized data.

From docs:
```
The key name under which all entities of this type will be listed in the normalized response. Must be a string name.
```

In your case you have movies entity for `movies`, `movie`, `myList` normalizers, so if you use `movies` key for all places, you will get similar normalized structure and you can the same action to merge all to data state.

Let's discuss tomorrow with examples.

Reply:
what I want to get in normalized response:

<img width="1556" alt="Screenshot 2020-03-25 at 19 34 31" src="https://user-images.githubusercontent.com/610948/77567360-b7410a00-6ecf-11ea-9a1c-7449ce2e6a1d.png">

Reply:
I have already fixed it but it would be nice to discuss it on the daily.

Reply:
Done